__bass.py

```diff
@@ -24,11 +24,13 @@ def gen_script():
                        .check_output(['bash', '-c', command])
                        .split(divider, 1))
     new_env = new_env.lstrip().splitlines()
-
+    
+    new_env = [line for line in new_env if '{' not in line and '}' not in line]
+    
     old_env = dict([line.split('=', 1) for line in old_env])
     new_env = dict([line.split('=', 1) for line in new_env])
 
-    skips = ['PS1', 'SHLVL', 'XPC_SERVICE_NAME']
+    skips = ['PS1', 'SHLVL', 'XPC_SERVICE_NAME', 'PATH']
```

Comment:
Line 50 was doing it for all variables instead of just PATH, which broke several other variables. PATH wasn't that important for me so I just left it out. If and when I get a chance to work on this again, I can try to add it back.